clifford/_multivector.py

```diff
@@ -739,6 +743,50 @@ def normalInv(self, check=True) -> 'MultiVector':
         """
         return self._pick_inv(fallback=False if check else None)
 
+    def hitzer_inverse(self):
+        """
+        Performs the inversion operation as described in the paper :cite:`Hitzer_Sangwine_2017`
+        """
+        tot = sum(self.layout.sig != 0)
+        if tot <= 5:
+            if tot == 1:
+                # Equation 4.3
+                mv_invol = self.gradeInvol()
+                numerator = mv_invol
+                denominator = (self * mv_invol)[()]
+            elif tot == 2:
+                # Equation 5.5
+                mv_conj = self.conjugate()
+                numerator = mv_conj
+                denominator = (self * mv_conj)[()]
+            elif tot == 3:
+                # Equation 6.10
+                mv_reverse = ~self
+                mv_invol = self.gradeInvol()
+                mv_conj = self.conjugate()
+                numerator = (mv_reverse * mv_invol * mv_conj)
+                denominator = (self * mv_conj * mv_invol * mv_reverse)[()]
```

Comment:
There is no 6.10...

I'd recommend the more symmetrical 6.5:
![image](https://user-images.githubusercontent.com/425260/86131767-f8077b00-badd-11ea-8ebc-c485211a5504.png)

Reply:
It would be nice to get all of these in the form `denominator = (self * numerator)[()]`